firmware/shared/controls/tvFactor.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include <vector>
+#include <algorithm>
+#include <tuple>
+#include "../util/mappers/lookup_table.h"
+
+namespace ctrl{
+
+template <typename T>
+T EnableTorqueVectoring(T steering_angle, T torque_vectoring_switch) {
+    if (torque_vectoring_switch > 0)
+        return steering_angle;
+    else return 0;
+}
+
+template <typename T>
+T CreateTorqueVectoringFactor(T steering_angle) {
+    T absolute_steering_angle = std::abs(steering_angle);
+
+    const float steering_angle_lookup_table_data[][2] = {
```

Comment:
since these 3 variables related to the lookup table are constant for each function call, it is more efficient to make them static variables. I''d like you to learn why and make this change